scalableminds/webknossos
# allow uploading nml for public dataset of different orga

### URL of deployed dev instance (used for testing):
- https://___.webknossos.xyz

### Steps to test:
- create second organization, set a dataset to public
- download an nml for that dataset, re-upload (as user of non-owning orga)
- should work
- uploading nml for non-public dataset of other orga should still be forbidden

### Issues:
- fixes #3922 

------
